packages/block-library/src/classic/edit.js

```diff
@@ -50,12 +50,20 @@ export default class ClassicEdit extends Component {
 	}
 
 	componentDidUpdate( prevProps ) {
-		const { clientId, attributes: { content } } = this.props;
-
+		const { clientId, attributes } = this.props;
 		const editor = window.tinymce.get( `editor-${ clientId }` );
+		let { contentRaw } = attributes;
+
+		// Strip white space before comparing to avoid resetting the editor content when not necessary;
```

Comment:
What kind of whitespace is different?

Reply:
The only differences I'm seeing are in some `<p>\u00a0</p>` and some `;` missing/being added to ends of inline style attributes.